site/docs/sql/00-intro.md

```diff
@@ -0,0 +1,141 @@
+---
+title: SQL Introduction
+description: Introduction to Hazelcast Jet SQL features.
+---
+
+Hazelcast Jet SQL service executes distributed SQL statements over
+Hazelcast collections and external data sets.
+
+It allows you to specify Jet jobs using the well-known SQL language.
+
+**Note:** _The service is in beta state. Behavior and API might change in
+future releases. Binary compatibility is not guaranteed between minor or
+patch releases._
+
+## Overview
+
+In the first release Jet SQL supports following features:
+
+- SQL Queries over [Apache Kafka topics](05-kafka-connector.md) and
+[Files (local and remote)](04-files-connector.md)
```

Comment:
Sorry if I am mistaken, but AFAIU from the SQL PR we support only IMap and Kafka in this release, but not files. Is this the case?